document/cloud-client/batch_parse_form.py

```diff
@@ -0,0 +1,96 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+# [START documentai_batch_parse_form]
```

Comment:
Oh oops should all the region tags have a `_beta` appended to the end? 
```suggestion
# [START documentai_batch_parse_form_beta]
```

@telpirion

Reply:
Up to you: what's the preferred convention?

Reply:
Convention is to rename the file `batch_parse_form_beta.py` and add `_beta` to the end of the region tag. 

Only so that when this goes GA, we don't break docs while we add new GA files and beta files can remain untouched. More of a headache saver than anything else.

Reply:
Done.